src/Resources/Cuberite.rc

```diff
@@ -16,21 +16,23 @@ BEGIN
 	2 "A lightweight, fast and extensible game server for Minecraft"
 END
 
-#define VERSION 0,0,13,37
-#define VERSION_STRING "0.0.13.37"
+#define VERSION 1,3,3,7
+#define VERSION_STRING "1.3.3.7"

```

Comment:
Does this change have any affect? I'm guessing not, but it is a major version change so potentially this could mess up people's VS?

Reply:
It's just what shows up in the properties dialog in Explorer, and I don't think anyone/anything else is using the exe metadata.

![Dialog](https://user-images.githubusercontent.com/2939130/105772757-936cab80-5f5a-11eb-8899-46a999e3b963.png)